src/transformers/training_args.py

```diff
@@ -153,7 +160,11 @@ class TrainingArguments:
     do_eval: bool = field(default=False, metadata={"help": "Whether to run eval on the dev set."})
     do_predict: bool = field(default=False, metadata={"help": "Whether to run predictions on the test set."})
     evaluate_during_training: bool = field(
-        default=False,
+        default=None,
+        metadata={"help": "Run evaluation during training at each logging step."},
+    )
+    evaluation_strategy: str = field(
+        default="no",
```

Comment:
I think we had a way to parse a "string" into an enum automatically in hf_parser at some point?

Reply:
Yeah but then something in Trainer complains the enum does not convert to json so I removed it for now. Will investigate more tomorrow.